.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Desolace.lua

```diff
@@ -1367,6 +1464,9 @@ root(ROOTS.Zones, m(KALIMDOR, {
 					["timeline"] = { ADDED_4_0_3 },
 					["races"] = ALLIANCE_ONLY,
 					["groups"] = {
+						objective(1, {	-- 0/6 Portals Banished
```

Comment:
Might need to check this one, I vaguely remember having to first interact with a Demon Portal object such as this: https://www.wowhead.com/cata/object=177399/demon-portal in order to spawn the guardian to kill it. In cases like this, we want to link the portal object first so that the player knows what they need to do.